pkg/storage/redisProvider.go

```diff
@@ -268,32 +206,11 @@ func (provider *Redis) Init() error {
 
 // Reset method will reset or close provider
 func (provider *Redis) Reset() error {
-	if provider.reconnecting {
-		provider.logger.Sugar().Error("Impossible to reset the redis instance while reconnecting.")
-		return nil
-	}
-	provider.close()
+	_ = provider.inClient.Do(provider.ctx, provider.inClient.B().Flushall().Build())
```

Comment:
I think it is better to only flush the selected db instead of flushing the whole cluster
```suggestion
	_ = provider.inClient.Do(provider.ctx, provider.inClient.B().Flushdb().Build())
```

Reply:
You're right, thank you for pointing that @hussam-almarzoq 🙏